pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoaderMain.java

```diff
@@ -680,6 +702,38 @@ private void processParticipant(String studyGuid, Map<String, JsonElement> sourc
                                 answerDao);
                     }
 
+                    if (hasMedical) {
+                        LOG.info("Loading Prion medical survey");
+                        String activityCode = mappingData.get("MedicalSurvey").getAsJsonObject().get("activity_code").getAsString();
+                        ActivityInstanceDto instanceDto = dataLoader.createActivityInstance(sourceData.get(
+                                "MedicalSurvey"),
+                                userGuid, studyId,
+                                activityCode, createdAt,
+                                jdbiActivity,
+                                activityInstanceDao,
+                                activityInstanceStatusDao,
+                                jdbiActivityInstance);
+                        dataLoader.loadMedicalSurveyData(handle, sourceData.get("MedicalSurvey"),
+                                mappingData.get("MedicalSurvey"),
+                                studyDto, userDto, instanceDto, answerDao);
+                    }
+
+                    if (hasPrionConsent) {
+                        LOG.info("Loading Prion consent survey");
+                        String activityCode = mappingData.get("ConsentSurvey").getAsJsonObject().get("activity_code").getAsString();
```

Comment:
Looks like only diff between PrionConsent and any other cmi consent is that export file contains json element name as "ConsentSurvey" rather than "consentsurvey" . You can probably generate prion export file with element "consentsurvey" and use existing method. 
If you prefer to have as "ConsentSurvey"  thats fine too, its just a tool for data loading. Some code duplication is ok.